common/lib/container-definitions/src/runtime.ts

```diff
@@ -121,6 +121,9 @@ export interface IRuntime extends IDisposable {
  * and the Container has created a new ContainerContext.
  */
 export interface IContainerContext extends IDisposable {
+    /**
```

Comment:
I believe client is on 0.45 or 0.47 version of container-definitions. That means it will take another half a year for this change to actually take effect. Anything we can do to speed it up? @anthony-murphy  for help.

Reply:
Are all usages gone? once that's all cleaned up, and that change has propagated, which should be much faster as it is all runtime an below, we can remove it here. Old loaders will still present it, but that doesn't matter if it is unused.

Reply:
there is only one usage, which will be gone once IContainerRuntime.id is removed. so it should be fast